addOns/requester/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- The Request/Response view option will now be saved each time it is changed (Issue 6985).
```

Comment:
I've ordered the change notes this way in the hopes that this PR will not conflict with https://github.com/zaproxy/zap-extensions/pull/3405 once that one is merged.